content/sensu-go/6.1/observability-pipeline/observe-schedule/subscriptions.md

```diff
@@ -0,0 +1,158 @@
+description: "Sensu subscriptions are equivalent to topics in a traditional publish/subscribe system. With Sensu subscriptions, you can configure check requests in a one-to-many model for an entire group or subgroup of systems rather than a traditional one-to-one mapping of configured hosts or observability checks. Read this reference doc to learn how to configure subscriptions in Sensu."
+weight: 35
+version: "6.1"
+product: "Sensu Go"
+platformContent: false
+menu:
+  sensu-go-6.1:
+    parent: observe-schedule
+---
+
+Sensu uses the [publish/subscribe model of communication][1].
+The publish/subscribe model is powerful in ephemeral or elastic infrastructures, where the names and numbers of things change over time.
+
+Because Sensu uses the publish/subscribe model, you can write checks even if you don't know the specific names of the entities that should run the checks.
+Likewise, your entities do not need to know the specific names of the checks they should execute.
+The Sensu backend coordinates check execution for you by comparing the subscriptions you specify in your checks and entities to determine which entities should receive execution requests for a given check.
+
+The diagram below shows how Sensu coordinates check execution based on subscriptions.
+For example, the `check_cpu` check includes the `system` subscription.
+All three entities include the `system` subscription, so all three entities will execute the `check_cpu` check.
+However, only Entity A and Entity C will execute `check_sshd_process` &mdash; Entity B does not include the `linux` subscription required to execute `check_sshd_process`.
+
+{{< figure src="/images/subscriptions_venn.png" alt="Venn diagram example of Sensu check execution based on subscriptions" link="/images/subscriptions_venn.png" target="_blank" >}}
+<!-- Diagram source: https://app.lucidchart.com/lucidchart/invitations/accept/25d40e7e-7293-4189-84e0-ef8c3a4c8ec5 -->
+
+*<p style="text-align:center">Sensu check execution based on subscriptions</p>*
+
+Sensu subscriptions are equivalent to topics in a traditional publish/subscribe system.
+Sensu entities become subscribers to these topics via the strings you specify with the agent `subscriptions` flag.
+Sensu checks have a `subscriptions` attribute, where you specify strings to indicate which subscribers will execute the checks.
+For Sensu to execute a check, the check definition must include a subscription that matches the subscription of at least one Sensu entity.
+
+As loosely coupled references, subscriptions avoid the fragility of traditional host-based monitoring systems.
+Subscriptions allow you to configure check requests in a one-to-many model for entire groups or subgroups of entities rather than a traditional one-to-one mapping of configured hosts or observability checks.
+
+## Configure subscriptions
+
+For Sensu to execute a check, the check definition must include a subscription that matches a subscription for at least one Sensu entity.
```

Comment:
Not sure if this is intentional or not, but this is sentence is an exact copy of a sentence 1 or 2 paragraphs above.

Reply:
I did leave it that way intentionally, but I can rephrase it so it's not distracting.